fuzzers/utils_fuzzer.c

```diff
@@ -24,8 +24,9 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
   ns = malloc(size+1);
   memcpy(ns, data, size);
   ns[size] = '\0';
-   
-  crm_parse_interval_spec(ns);
+
+  guint result;
```

Comment:
C99 (declarations have to be before code)

Reply:
Thanks, done!